manifests/iso.pp

```diff
@@ -7,7 +7,14 @@
 
   $target_file = "${mrepo::src_root}/iso/${name}"
 
-  staging::file { $name:
+  file { "${mrepo::src_root}/iso":
+    ensure  => directory,
+    owner   => $mrepo::user,
+    group   => $mrepo::group,
+    mode    => "0644",
+  }
+
+  -> staging::file { $name:
```

Comment:
The staging module is deprecated. Could you implement this with https://forge.puppet.com/puppet/archive ?

Reply:
ah, the diff trolled me, that was already present. You don't need to rework this (unless you want to).